src/components/views/Symptoms.vue

```diff
@@ -1,82 +1,69 @@
+        <ul v-for="(symptom, index) in symptoms.data" 
+            :key="index" 
+            class="main-ul">
+          <li>
+            <font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>{{ symptom }}
+          </li>
+        </ul>
+      </div>
     </div>
+
     <div class="row p-3">
-      <div class="col-sm-12 col-md-10 col-lg-10 offset-1">
-        <h2>Pre-existing conditions</h2>
-        <ul class="main-ul">
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Cardiovascular disease</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Diabetes</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Chronic respiratory disease</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Hypertension</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Lung, Heart, or Kidney disease</li>
-          </ul>
+      <div  v-for="(conditions, index) in symptomsJson.conditions" 
+            :key="index" 
+            class="col-sm-12 col-md-10 col-lg-10 offset-1">
+        <h2>{{ conditions.type }}</h2>
+        <ul v-for="(condition, index) in conditions.data" 
+            :key="index" 
+            class="main-ul">
+          <li>
+            <font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>{{ condition }}
+          </li>
+        </ul>
       </div>
     </div>
+    
     <div class="row p-3">
-      <div class="col-sm-12 col-md-10 col-lg-10 offset-1">
-        <h2>Protection mechanisms</h2>
-        <ul class="main-ul">
-          <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Wash your hands often, for at least 20 seconds each time, with soap and water.</li>
-          <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Use an alcohol-based sanitizer with at least 60% alcohol if you don't have to soap and water handy.</li>
-          <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Avoid people who are sick.</li>
-          <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Don’t touch your eyes, nose, or mouth unless you’ve just washed your hands.</li>
-          <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Regularly clean and disinfect surfaces that you touch a lot.</li>
+      <div v-for="(preventions, index) in symptomsJson.preventions" 
+          :key="index"
+          class="col-sm-12 col-md-10 col-lg-10 offset-1">
+        <h2>{{ preventions.type }}</h2>
+        <ul v-for="(prevention, index) in preventions.data" 
+            :key="index"
+            class="main-ul">
+          <li>
+            <font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>{{ prevention }}
+          </li>
         </ul>
       </div>
     </div>
   </div>
 </template>
 
 <script>
+import json from '@/json/symptoms.json'
```

Comment:
The whole idea was to read the data from the json and resolve using a async call. We store this data in vuex store by dispatching an action to get this data.

We can then utilise vuex reactive system to force DOM updates

E.g 

```

// vuex

actions: {
    async getsymptoms () {
        const data = await fetch('json/symptoms.json')
        commit('setsymptoms', data)
    }
}

Include the mutation and getter

//vue componentnt
<template>
.....
display symptoms
</template>
<script>
......
computed: {
    symptoms () {
        return this.$store.state.symptoms
    }
}
created () {
    this.$store.dispatch('getsymptoms')
}
</script
```

Reply:
Benfits:
- We can modify our fetch URL to get data from external source incase we decide to retrieve data from data for example without affecting the underlying implementation
- Easier to write unit tests